src/app/(tabs)/movies/[id]/page.tsx

```
@@ -11,20 +52,184 @@ const MovieDetails = () => {
+                      itemBadge:
+                        'bg-ui-transparent border-ui-600 hover:border-ui-400 border-1',
+                    }}
+                  />
+                </div>
+                {/*Status */}
+                <div className="flex flex-col gap-2">
+                  {renderMovieInfoTitle('Status')}
+                  <Badge
+                    className={cn(
+                      'text-base-white rounded-full px-2',
+                      'released' === 'released' ? 'bg-brand-600' : 'bg-ui-600',
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)